cordova-lib/src/plugman/registry/registry.js

```diff
@@ -218,10 +205,18 @@ module.exports = {
 
 /**
  * @method initSettings
+ * @param {Boolean} using npm registry
  * @return {Promise.<Object>} Promised settings.
  */
-function initSettings() {
+function initSettings(npm) {
     var settings = module.exports.settings;
+    var registryURL = 'http://registry.cordova.io';
+
+    //if npm is true, use npm registry
+    if(npm) {
+        registryURL = 'http://registry.npmjs.org';
+    }
+
```

Comment:
Ok, fair enough - I get some people don't like `?:`. The usual alternative is:

    var registryURL;
    if (npm) {
        registryURL = 'http://registry.npmjs.org';
    } else {
         registryURL = 'http://registry.cordova.io';
    }

I know this is slightly more verbose, but I think it is cleaner code than assigning something, then immediately assigning something else if a condition is true. That said, it's only a minor nit so I won't complain very loudly if you leave it as is :).